.travis.yml

```diff
@@ -205,3 +218,9 @@ script:
       bjam arm-tests warnings-as-errors=on ssl=$ssl crypto=$crypto variant=test_arm $toolset target-os=linux link=static testing.launcher="sudo cp -R bin rootfs/; sudo chroot rootfs";
     fi'
   - cd ..
+
+# test build for iOS, using x86_64 to avoid the need to build boost from source for arm
```

Comment:
oh, I iphonesim is not arm, so isn't this comment misleading? I'm not sure I understand what you mean.